update-contracts.sh

```diff
@@ -21,7 +21,7 @@ if [[ "$AUTOCOMMIT" == "true" ]]; then
   git commit -m "Auto-updating from push to augur-core#${BRANCH} (${COMMIT})"
 
   case $BRANCH in
-    v+([0-9]).+([0-9]).+([0-9])?(-+([0-9])))
+    master | v+([0-9]).+([0-9]).+([0-9])?(-+([0-9])))
       if [[ "$BRANCH" == "$TAG" ]]; then
```

Comment:
We need to remove this condition, and basically always push the tags if we are on master or the branch name matches the release tag expressions.